client/src/main/java/de/hellfish/ngrok/client/TestService.java

```diff
@@ -0,0 +1,58 @@
+package de.hellfish.ngrok.client;
+
+import lombok.extern.slf4j.Slf4j;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.PrintWriter;
+import java.net.ServerSocket;
+import java.net.Socket;
+import java.nio.charset.StandardCharsets;
+
+@Slf4j
+public class TestService {
```

Comment:
Not needed.

Reply:
Actually, I need this class. It is test Service, which i will proxy with my app.